app/src/lib/get-os.ts

```diff
@@ -83,3 +83,13 @@ export const isWindowsAndNoLongerSupportedByElectron = memoizeOne(
 export const isMacOSAndNoLongerSupportedByElectron = memoizeOne(
   () => __DARWIN__ && systemVersionLessThan('10.15')
 )
+
+export function isOSNoLongerSupportedByElectron(): boolean {
+  if (__DARWIN__) {
+    return isMacOSAndNoLongerSupportedByElectron
+  } else if (__WIN32__) {
+    return isWindowsAndNoLongerSupportedByElectron
+  } else {
+    return false
+  }
```

Comment:
```suggestion
return isMacOSAndNoLongerSupportedByElectron() || isWindowsAndNoLongerSupportedByElectron()
```

The above methods already check for OS so I think that is redundant and this method could simplified. (Drive-by review from my phone so warning formatting of the above suggestion is probably not ideal)